functions/src/plugins/merge.ts

```diff
@@ -257,56 +257,27 @@ export class MergeTask extends Task {
    */
   async getPendingReviews(context: Context, pr: Github.PullRequestsGetResponse): Promise<number> {
     const {owner, repo} = context.repo();
-    // we only want reviews with state: PENDING, APPROVED, CHANGES_REQUESTED, DISMISSED
-    // we ignore comments because they can be done after a review was approved / refused
-    // also anyone can add comments, it doesn't mean that it's someone who is actually reviewing the PR
-    const query = `
-      reviews(last: 50, states: [PENDING, APPROVED, CHANGES_REQUESTED, DISMISSED]) {
-        nodes {
-          authorAssociation
-          author {
-            ... on User {
-              userId: id
-            }
-          }
-          state
-          createdAt
-        }
-      }
-      reviewRequests(last: 10) {
-        nodes {
-          requestedReviewer {
-            ... on User {
-              userId: id
-            }
-            ... on Team {
-              teamId: id
-            }
-          }
-        }
-      }
-    `;
-
-    const prData = await this.queryPR<ReviewQuery>(context, query, {
-      owner,
-      repo,
-      number: pr.number
-    });
-
-    const reviews = prData.reviews.nodes
-    // order by latest review first
-      .sort((review1, review2) => new Date(review2.createdAt).getTime() - new Date(review1.createdAt).getTime());
+    const reviews = ((await context.github.pullRequests.listReviews({owner, repo, number: pr.number})).data as any as Review[])
+      // we only want reviews with state: PENDING, APPROVED, CHANGES_REQUESTED, DISMISSED
+      // we ignore comments because they can be done after a review was approved / refused
+      // also anyone can add comments, it doesn't mean that it's someone who is actually reviewing the PR
+      .filter(review => review.state !== REVIEW_STATE.Commented)
+      // we ignore reviews from people who aren't members of the repo
+      .filter(review => review.author_association !== AUTHOR_ASSOCIATION.None)
+      // order by latest review first
+      .sort((review1, review2) => new Date(review2.submitted_at).getTime() - new Date(review1.submitted_at).getTime());
 
     // the list of requested reviewers only contains people that have been requested for review but have not
     // given the review yet. Once he does, he disappears from this list, and we need to check the reviews
-    const reviewRequests = prData.reviewRequests.nodes.length;
-    const usersReviews: string[] = [];
+    const reviewRequests =(await context.github.pullRequests.listReviewRequests({owner, repo, number: pr.number})).data.users;
```

Comment:
also please document that this is where we filter out group reviews and consider only reviews from individuals..